src/Analyzers/Core/CodeFixes/GenerateConstructor/GenerateConstructorHelpers.cs

```diff
@@ -117,4 +123,222 @@ private static bool IsCompatible<TExpressionSyntax>(
 
         return true;
     }
+
+    public static async Task<
```

Comment:
this is just a move of the shared logic.  It stayed the same except that it returns the values as it cannot directly assign into the destination fields of a type as it used to.

Reply:
all the logic to the end of the file is just a move and is unchange.d